src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -747,6 +747,33 @@ export class PostgresStorageAdapter {
       });
   }
 
+  schemaUpgrade(className, schema) {
+    debug('schemaUpgrade', { className, schema });
+
+    return this._client.tx('schemaUpgrade', t => {
+      return t.any('SELECT column_name FROM information_schema.columns WHERE table_name = $<className>', { className })
+        .then(columns => {
+          const currentColumns = columns.map(item => item.column_name);
+          const newColumns = Object.keys(schema.fields);
+
+          const columnsToCreate = newColumns.filter(item => currentColumns.indexOf(item) === -1);
+
+          const addFields = [];
+
+          columnsToCreate.forEach(fieldName => {
+            const type = schema.fields[fieldName];
+            addFields.push(this.addFieldIfNotExists(className, fieldName, type));
+          });
+
+          if (!addFields.length) {
```

Comment:
Inefficient use of promises here. Can simply skip returning anything for the same effect.